.github/workflows/compose_screenshot_test.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   sceenshot_test:
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
@isaacnguyen0809 can we use Linux? What's the benefit of running them Windows? My concern is that GitHub Actions Windows are more expensive => will provide less instances for open-source projects

Reply:
Got it, no rush^ I marked the PR with "keep" so the CI won't close it. Take your time :)

Reply:
The screenshot tests must work for all platforms because we have many contributors with different machines